classes/test.php

```diff
@@ -304,7 +302,16 @@ function ($value) use ($test, & $methodName) {
             ->setHandler('small', function () use ($tagHandler) {
                 $tagHandler('small');
             })
-        ;
+            ->setHandler('covers', function ($value) use ($test) {
+                foreach (annotations\extractor::toArray($value) as $target) {
+                    // We only care about the classname to compute the real
+                    // code coverage score, nothing else.
+                    if (0 !== preg_match('/^([^:]+)(::)?/', $target, $matches)) {
+                        $test->setTestedClassName($matches[1]);
+                        $this->breakRelationBetweenTestSuiteAndSUT = false;
+                    }
```

Comment:
What to do to raise a warning? An option is to do nothing and to add a note in the `README.md`.